javascript/react/fundamentals/the-react-component.md

```diff
@@ -36,27 +36,47 @@ In **React** a `component` represents a fundamental *class* you can work with to
 
 These `component`s behave like simple functions, they can take input arguments called `props` and will return what should be *rendered*.
 
-The *rendered* output needs to be specified inside a function called `render()`.
+If you want to create a simple component, you can use a JavaScript function like this:
 
-To create a component you can `extend` the `ES6`[1] class `React.Component`:
+```jsx
+// functional component
+function Enki(props) {
+  return <p>Enki aka Ea</p>;
+}
 ```
+
+At the same time, you can also `extend` the `ES6`[1] class `React.Component` to create a component. In this case, the *rendered* output needs to be specified inside a function called `render()`:
+
+```jsx
+// class component
 class Enki extends React.Component {
   render() {
     return <p>Enki aka Ea</p>;
   }
 }
 ```
 
-You can pass dynamic values to a component via `props` which will be discussed next. In addition, you can use other components when `render`ing.
+The differences between functional and class components will be discussed at the end of this workout, but for now know that React views both of the components defined above as being the same.
+
+You can pass dynamic values to a component via `props` which will be discussed next. 
 
 `Component` names should start with a capital letter such that you can differentiate them from **DOM tags**.
 
-An `element` can represent your `component` as well:
-```
+You can use other components when `render`ing, and an `element` can represent your `component` as well:
```

Comment:
this is ambiguous. what is `element`? a simple variable?  why is the `component` written as code? is that another variable? 

> instances of components can be assigned to JS variables
^ is this the takeaway?

Reply:
- `component` was just kept from the old content, I'll remove the code ticks
- `element` refers to a React element. I'll just use React element instead of `element`